agent/agent_test.go

```diff
@@ -1167,11 +1194,14 @@ task_groups:
         params:
           script: sleep 5
 `
-	s.setupRunTask(projYml)
-	s.tc.taskConfig.Task.TaskGroup = taskGroup
+	p := &model.Project{}
+	_, err := model.LoadProjectInto(s.ctx, []byte(projYml), nil, "", p)
+	s.Require().NoError(err)
```

Comment:
It seems like replacing `setupRunTask` with `LoadProjectInto` + setting the project is unnecessary.